airbyte-integrations/connectors/source-cart/source_cart/source.py

```diff
@@ -44,10 +45,29 @@ def get_auth_header(self) -> Mapping[str, Any]:
 
 
 class SourceCart(AbstractSource):
+    def validate_config_values(func):
+        """Check input config values for check_connection and stream functions. It will raise an exception if there is an parsing error"""
+
+        @wraps(func)
+        def decorator(self_, *args, **kwargs):
+            for arg in args:
+                if isinstance(arg, Mapping):
+                    pendulum.parse(arg["start_date"])
```

Comment:
is the point of this line to raise an exception if it's not parsable as a date? if so a comment would be super helpful, maybe even wrapping it in a try/except with a readable error message